cms/static/cms/js/modules/cms.toolbar.js

```diff
@@ -454,7 +454,7 @@ var CMS = window.CMS || {};
             _show: function _show(opts) {
                 var speed = opts && opts.duration !== undefined ? opts.duration : this.options.toolbarDuration;
                 var debugHeight = $('.cms-debug-bar').height() || 0;
-                var toolbarHeight = $('.cms-toolbar').height() + 10;
```

Comment:
it was needed to speed up the performance of the rendering while dragging in structureboard, because the box-shadow that leaked from the toolbar was causing a big chunk of frames recomposing :(